src/QueryEditor/util.ts

```diff
@@ -311,3 +328,36 @@ export function migrateMeasurementQuery(query: MeasurementQuery): MeasurementQue
 
   return measurementQuery
 }
+
+// semverCompare compares two semver strings and returns true if a is less than b
+export function semverCompare(a: string, b: string): boolean {
+  // remove leading v
+  if (a.startsWith('v')) {
+    a = a.substring(1)
+  }
+  if (b.startsWith('v')) {
+    b = b.substring(1)
+  }
+  const aParts = a.split('.')
+  const bParts = b.split('.')
+
+  for (let i = 0; i < aParts.length; i++) {
+    if (i >= bParts.length) {
+      return false
+    }
+    console.log(aParts[i], bParts[i])
+    const aPart = parseInt(aParts[i], 10)
+    const bPart = parseInt(bParts[i], 10)
+    console.log(aPart, bPart)
+
+    if (aPart < bPart) {
+      return true
+    }
+
+    if (aPart > bPart) {
+      return false
+    }
+  }
+
+  return aParts.length < bParts.length
+}
```

Comment:
can you rework the semverCompare to reflect the standard compare function in javascript?

function compare(a, b){return a — b}

Return greater than 0 if a is greater than b
Return 0 if a equals b
Return less than 0 if a is less than b